doc/classes/GridMap.xml

```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="GridMap" inherits="Spatial" category="Cpre" version="3.0.alpha.custom_build">
+	<brief_description>
+		Node for 3D tile-based maps.
+	</brief_description>
+	<description>
+		Node for 3D tile-based maps.
+		
+		GridMaps use a [MeshLibrary] which contain a list of tiles (meshes with materials plus optional collisions and extra elements) which can be used to create grid-based maps in 3D.
+
+		A GridMap contains a collection of Cells. Each grid Cell refers to a [MeshLibrary] item. All Cells in the map have the same dimensions.
+
+		A GridMap is split into a sparse collection of Octants for efficient rendering and physics processing. Every Octant has the same dimensions and can contain several Cells.
```

Comment:
Remove caps on cells if they're just a concept. If it's some member variable, either link to it [member var_name] or use [code][/code]

Reply:
They are technically private inner classes. As far as the user is concerned they don't exist in code, so I assume the correct way is treat them as concepts (lowercase).